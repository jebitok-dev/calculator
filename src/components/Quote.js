const Quote = () => (
  <div className="centered">
    <h3>
      Mathematics is not about numbers, equations, computations, or algorithms: it
      is about understanding.
      <br />
      {' '}
      -William Paul Thurston
    </h3>
  </div>
);

export default Quote;
