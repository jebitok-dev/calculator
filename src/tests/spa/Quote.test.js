import renderer from 'react-test-renderer';
import React from 'react';
import Quote from '../../components/Quote';

it('should match the Quote Snapshot', () => {
  const quote = renderer.create(<Quote />).toJSON();
  expect(quote).toMatchSnapshot();
});
