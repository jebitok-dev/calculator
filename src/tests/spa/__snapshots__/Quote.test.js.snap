// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should match the Quote Snapshot 1`] = `
<div
  className="centered"
>
  <h3>
    Mathematics is not about numbers, equations, computations, or algorithms: it is about understanding.
    <br />
     
    -William Paul Thurston
  </h3>
</div>
`;
